Cargo.toml

```diff
@@ -35,9 +35,12 @@ transform-types = []
 
 # libm is required when building no_std
 libm = ["num-traits/libm"]
+#
+crevice_traits = ["crevice", "bytemuck"]
```

Comment:
I'm not sure this is necessary? You should be able to check for crevice, you might need to add a bytemuck feature to the crevice dependency if it's not a default feature.

Reply:
I `use bytemuck::Zeroable;` in `impl_crevice.rs`, so I think I need the dep to enable crevice_traits?

Reply:
I see, I thought crevice must have added bytemuck support via a feature, since they mention implementing bytemuck traits in their readme, but this doesn't seem to be the case.

Reply:
(just a drive-by following random threads as I do crate research)

If you want to "clever around" this and just allow deps to write `features = ["crevice"]`, you can:

- [rename the dependency](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html?highlight=rename,depende#renaming-dependencies-in-cargotoml) with e.g. `crevice_ = { package = "crevice", .. }`
- declare a `crevice = ["crevice_", "bytemuck"]` feature

It's a question of which hacky thing you want to do; rename the local crate name, or rename the feature.

-----

Alternatively, to eliminate the requirement on `bytemuck` for the impl you could

- Get `crevice` to implement `Default::default`, and use that (probably not a bad idea), or
- Name the padding fields in the macro expansion, and initialize the padding fields with `Default::default`.